packages/internet-header/src/components/post-internet-header/readme.md

```diff
@@ -17,7 +17,7 @@
 | `language`                | `language`                   | Initial language to be used. Overrides automatic language detection.                                                                                                                                                      | `"de" \| "en" \| "fr" \| "it" \| undefined`                               | `undefined`                            |
 | `languageCookieKey`       | `language-cookie-key`        | The header uses this cookie to set the language. Disables automatic language detection.                                                                                                                                   | `string \| undefined`                                                     | `undefined`                            |
 | `languageLocalStorageKey` | `language-local-storage-key` | The header uses this local storage key to set the language. Disables automatic language selection.                                                                                                                        | `string \| undefined`                                                     | `'swisspost-internet-header-language'` |
-| `languageSwitchOverrides` | `language-switch-overrides`  | Override the language switch links with custom URLs. Helpful when your application contains sub-pages and you would like to stay on subpages when the user changes language.                                              | `IAvailableLanguage[] \| string \| undefined`                             | `undefined`                            |
+| `languageSwitchOverrides` | `language-switch-overrides`  | Override the language switch links with custom URLs. Helpful when your application contains sub-pages, and you would like to stay on subpages when the user changes language.                                             | `IAvailableLanguage[] \| string \| undefined`                             | `undefined`                            |
```

Comment:
This is an uncommitted change from a PR that has been merged on main (not sure which one), but it appears everywhere, so let's fix it.